documentation/docs/scenarios/xsa-deploy/Readme.md

```diff
@@ -1,13 +1,13 @@
-# Build and Deploy SAP Fiori Applications on SAP HANA XS Advanced
+# Build and Deploy SAP Fiori Applications on SAP HANA Extended Application Services, Advanced Model
 
-Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result into an SAP Cloud Platform account in the Neo environment.
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result to SAP HANA extended application services, advanced model.
 
 ## Prerequisites
 
 * [Docker environment](https://docs.docker.com/get-started/)
-* All artifacts refereneced during the build are available either on Service Market Place or via public repositories
+* All artifacts referenced during the build are available either on Service Market Place or via public repositories.
 * You have set up Project “Piper”. See [guided tour](https://sap.github.io/jenkins-library/guidedtour/).
-* Docker image for xs deployment available. Due to legal reasons there is no pre-build docker image. How to create the docker image is explained [here](https://github.com/SAP/devops-docker-images/tree/master/xs-cli).
+* Docker image for xs deployment is locally available. Due to legal reasons there is no pre-build Docker image. How to create the Docker image is explained [here][devops-docker-images-xs-deploy-readme].
```

Comment:
> Docker image (...) locally available.

Well. In fact it is not available. What we have is a docker script which can be used in order to create a docker image locally.

Reply:
This is a prerequisite, i.e. the image should be _locally available_ to run the scenario. In order to have it _locally available_ build instructions are linked. Could you give me an alternative, because _for me_ it's clear ... but usually that doesn't mean it's clear for everybody :) Maybe @SarahNoack can help us out here.

Reply:
A ok, in the UI during review the headline `Prerequisites` was not visible. Hence I missed that the bullet point is under a section `Prerequisites`. With that in mind it is OK.